src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+[output] 69zjIFUfApJiUt8gFmi-6-dcPe8/.terraform.lock.hcl to /Users/jsius/Sites/deco/terragrunt-config/environments/dev
+```
+
+Run `terragrunt plan`:
+```bash
+terragrunt plan
+```
+
+Terragrunt starts to run but pauses to ask for the inputs to the variables added to the __provider.tf__ file.
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] 2/02 13:32:08 Running command: terraform plan
+[output] var.newrelic_account_id
+[output]  Enter a value:
+[output] Interrupt received.
+```
+
+Exit the `terragrunt plan` process. In the environments/dev __terragrunt.hcl__ file, add an inputs block to provide the values for your New Relic account vairables:
+```hcl
+inputs = {
+	newrelic_personal_apikey = "NRAK-***" # Your New Relic account ID
+	newrelic_account_id = "12345" # Your New Relic account ID
+	newrelic_region = "US" # US or EU (defaults to US)
+}
+```
+
+Remove the New Relic account variables added to the provider.tf file in the previous steps. The updated terragrunt.hcl only has the include, terraform, and inputs block
+```hcl
+include {
+	path= find_in_parent_folders()
+}
+
+terraform {
+	source = "../../src"
+}
+
+inputs = {
+	newrelic_personal_apikey = "NRAK-***" # Your New Relic account ID
+	newrelic_account_id = "12345" # Your New Relic account ID
+	newrelic_region = "US" # US or EU (defaults to US)
+}
+```
+
+Now, when you run `terragrunt plan`, Terragrunt will provide the input block's values, no longer prompting the command line for values.  Run `terragrunt plan`:
+
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] An execution plan has been generated and is shown below.
+[output] Resource actions are indicated with the following symbols:
+[output]  + create
+[output]
+[output] Terraform will perform the following actions:
+[output]
+[output]  # newrelic_alert_policy.DemoPolicy will be created
+[output]  + resource "newrelic_alert_policy" "DemoPolicy" {
+[output]    + account_id     = (known after apply)
+[output]    + id         = (known after apply)
+[output]    + incident_preference = "PER_POLICY"
+[output]    + name        = "My Demo Policy"
+[output]   }
+[output]
+[output] Plan: 1 to add, 0 to change, 0 to destroy.
+[output]
+[output] ------------------------------------------------------------------------
+[output]
+[output] Note: You didn't specify an "-out" parameter to save this plan, so Terraform
+[output] can't guarantee that exactly these actions will be performed if
+[output] "terraform apply" is subsequently run.
+```
+
+Run `terragrunt apply`. After Terraform finishes process, check your New Relic account for the new Demo Policy in your New Relic account.
```

Comment:
```suggestion
Run `terragrunt apply`. After Terraform finishes process, check your New Relic account for the new Demo Policy.
```

Since you've already stated "check your New Relic account", I removed the repeated "in your New Relic account" bit.